coalib/files/Fileproxy.py

```diff
@@ -0,0 +1,89 @@
+from coala_utils.decorators import generate_eq
```

Comment:
We currently create inits for all modules. This can be brought up in a new issue? If we're removing it here, let's remove it everywhere.
Example: https://github.com/coala/coala/blob/master/coalib/processes/__init__.py